ee/backup/run.go

```diff
@@ -111,6 +112,10 @@ $ dgraph restore -p . -l /var/backups/dgraph -z localhost:5080
 	}
 
 	flag := Restore.Cmd.Flags()
+	flag.StringVar(&opt.compression, "badger.compression", "snappy",
```

Comment:
This is the third place in the code where this exact description has been duplicated. Perhaps we could extract it out elsewhere.